cadence/contracts/LendingComptroller.cdc

```diff
@@ -312,7 +310,7 @@ pub contract LendingComptroller {
         /// @Param repayUnderlyingAmountScaled - Repay amount scaled by 1e18
         /// @Return error code
         ///
-        pub fun repayAllowed(
+        access(all) fun repayAllowed(
```

Comment:
`view`?

Reply:
Cannot be made view as non-view operation performed in `callerAllowed()`